stylesheet.css

```diff
@@ -0,0 +1,683 @@
+  input[type="checkbox"] {
+    appearance: none;
+  }
+
+  /* Menu button on mobile END */
+  /* Header END */
+
+  .bg-container {
+    grid-area: 2 / 1 / 3 / 2;
+    background-image: url("./assets/images/bird/b1.png");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: cover;
+  }
+
+  /* Works */
+  .works {
+    padding-inline: 1em;
+    padding-block: 1rem;
+  }
+
+  .section-anatomy img,
+  .section-ch img {
+    max-inline-size: 100%;
+    block-size: auto;
+    object-fit: contain;
+  }
+
+  /* Animations images mobile */
+  .section-anatomy img:not(:first-child),
+  .section-ch img {
+    animation: fade-in linear;
+    animation-timeline: view();
+    animation-range-start: 50px;
+    animation-range-end: 300px;
+  }
+
+  .content-container {
+    padding-block: 2rem;
+  }
+
+  .section-anatomy,
+  .section-ch {
+    display: flex;
+    flex-direction: column;
+    margin-block: 1rem;
+  }
+
+  .section-anatomy h2,
+  .section-ch h2 {
+    align-self: flex-end;
+    margin-block: 1rem;
+  }
+
+  .anatomy-images-wrapper,
+  .ch-images-wrapper {
+    margin-block: 1rem;
+  }
+
+  .anatomy-images-wrapper img,
+  .ch-images-wrapper img {
+    padding-block: 1rem;
+    padding-inline: 0rem;
+  }
+
+  .anatomy-images-wrapper img:not(:last-child),
+  .ch-images-wrapper img:not(:last-child) {
+    margin-block-end: 1rem;
+  }
+
+  /* Works END */
+
+  footer {
+    grid-area: 3 / 1 / 4 / 2;
+    padding-block: 0rem;
+    text-align: center;
+    align-self: end;
+  }
+
+  /* Contact Form */
+  .form-wrapper {
+    min-width: 70%;
+  }
+
+  /* Contact Form  END*/
+
+  /* About Me */
+  .about {
+    min-height: 100vh;
+    padding-inline: 1rem;
+    padding-block: 1rem;
+  }
+
+  .about-me-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .about-me-description {
+    padding-block: 5rem 5rem;
+  }
+
+  .about-me-description h2 {
+    text-transform: capitalize;
+    padding-inline-start: 1rem;
+  }
+
+  .about-me-description p {
+    line-height: 1.8rem;
+    letter-spacing: 0.1rem;
+    padding-inline: 1rem;
+    text-wrap: pretty;
+    word-break: break;
+  }
+
+  .about-me-description p:not(:last-child) {
+    margin-block: 0 1rem;
+  }
+
+  .about-me-wrapper img {
+    block-size: auto;
+    max-inline-size: 100%;
+    object-fit: cover;
+    display: block;
+    padding-block: 1rem 1rem;
+  }
+
+  figcaption {
+    font-size: 0.8rem;
+    text-align: center;
+    margin-block-end: 5rem;
+  }
+
+  .header-social-media {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .social-media {
+    display: flex;
+    flex-direction: row;
+    gap: .6rem;
+    padding-inline-start: 1rem;
+    margin-block-end: 1rem;
+  }
+
+  .social-media img {
+    padding-bottom: 10px;
+    padding-top: 1px;
+  }
+
+  /* About Me END */
+}
+
+/* Desktop and Larger */
+@media (min-width: 48.06rem) {
```

Comment:
De ce acest media query atât de precis în `rem`?

Reply:
Initial am inceput sa folosesc media queries breakpoints cu operatori logici si ceva nu mergea cum trebuie la anumite marimi.  Si am mers pe standard media queries breakpoints Mobile devices – 320px — 480px.
iPads, Tablets – 481px — 768px.
Small screens, laptops – 769px — 1024px.
Desktops, large screens – 1025px — 1200px.
Extra large screens, TV – 1201px, and more.